v2/src/components/reusableSnippets/angularUIImplementation.tsx

```diff
@@ -5,7 +5,6 @@ export default function AngularUIImplementation(props: PropsWithChildren<{}>) {
   return (
     <Question
       question="What type of UI are you using?"
-      persistentId="angular-ui"
```

Comment:
why did this change?

Reply:
This was breaking the build and I checked the latest code in the branch and the build was failing as well so I went ahead and fixed it.

I can undo it if you want.

Reply:
As far as I can tell that component is not actually used in the mdx pages (just imported). 
You can go ahead with this change. I'll remove the component when we merge the v3 branch

Reply:
Thanks for the confirmation!